src/api/checkout/checkout_payments/v1/_idpayment_check.xml.tpl

```diff
@@ -0,0 +1,34 @@
+<policies>
+    <inbound>
+      <base />
+      <!-- Check idPayment in IO env- START -->
+      <set-header name="x-functions-key" exists-action="override">
+        <value>{{pagopa-fn-checkout-key}}</value>
+      </set-header>
+      <set-backend-service base-url="{{pagopa-fn-checkout-url}}" />
+      <!-- Check idPayment in IO env - END -->
+    </inbound>
+    <outbound>
+      <base />
+      <choose>
+        <when condition="@(context.Response.StatusCode == 404)">
+          <!-- Check idPayment in pagoPA env- START -->
+          <set-variable name="codice_contesto_pagamento" value="@(context.Request.MatchedParameters["codice_contesto_pagamento"])" />
+          <send-request ignore-error="true" timeout="10" response-variable-name="response" mode="new">
+            <set-url>$"{{pagopa-appservice-proxy-url}}/api/v1/payment-activations/${context.Variables["codice_contesto_pagamento"]}"</set-url>
```

Comment:
`pagopa-appservice-proxy-url` is set to io pagopa-proxy right ?

Reply:
yes, is pagopa-proxy deployed in pagoPA